css/portal-dashboard/all-responses-popup/selected-student-list.less

```diff
@@ -0,0 +1,198 @@
+@import "../variables";
+
+.spotlightListDialog {
+  border-radius: 8px;
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.35);
+  background-color: white;
+  position: absolute;
+  top: 25px;
+  left: 25px;
+  right: 25px;
+  bottom: 25px;
+  z-index: 1000;
+  box-shadow: 0 0 0 8000px rgba(0,0,0,0.35);
+
+  .spotlightListHeader {
+    height: 50px;
+    border-radius: 8px 8px 0 0;
+    background-color: @cc-teal-light4;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    .headerLeft {
+      margin-left: 20px;
+
+      .spotlightIcon {
+        width: 32px;
+        height: 32px;
+        object-fit: contain;
+        fill: @cc-teal;
+      }
+    }
+
+    .headerCenter {
+      display: flex;
+      flex-direction: row;
+      flex: 1;
+      align-items: center;
+      width: 400px;
+      height: 100%;
+      font-size: 16px;
+
+      .assignmentContainer {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        margin: auto;
+
+        .assignmentIcon {
+          fill: @cc-teal;
+        }
+
+        .title {
+          flex: 1;
+          margin-left: 5px;
+          font-weight: bold;
+        }
+      }
+    }
+
+    .headerRight {
+      margin-right: 20px;
+      .closeIcon {
```

Comment:
At some point it would be nice to factor these x close buttons into a single component.  I don't think we need to do it here, but I think we now have three redundant x close buttons in the app that all have the same styling other than some color.  We could eventually roll that into a single component for easier maintainability and DRY code.